share/picongpu/benchmarks/SPEC/include/picongpu/param/memory.param

```diff
@@ -112,4 +112,21 @@ namespace picongpu
      */
     constexpr bool fieldTmpSupportGatherCommunication = false;
 
+    // struct ParticleFrameMapping : llama::mapping::BindAoS<false>
+    struct ParticleFrameMapping : llama::mapping::BindSoA<false>
+    // struct ParticleFrameMapping : llama::mapping::BindAoSoA<16>
+    // struct ParticleFrameMapping : llama::mapping::BindAoSoA<32>
+    // struct ParticleFrameMapping : llama::mapping::BindAoSoA<64>
+    {
+        inline static constexpr bool splitVector = false;
+    };
```

Comment:
For:

> Find a better way to choose which memory layout to use in the param files

I am using a quoted metafunction (`ParticleFrameMapping`) now which has an additional flag whether the `Vector`s should be split up into their components. That is good enough for most cases I guess. It can cover all LLAMA mappings which only depend on static/compile-time information.